sui/token_bridge/Move.toml

```diff
@@ -7,4 +7,4 @@ Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-fram
 Wormhole = { local = "../wormhole" }
 
 [addresses]
-token_bridge="0x0"
\ No newline at end of file
+token_bridge = "0xf8fdb4bacdec4faa9416d6ebdedd2231a09cc60a"
```

Comment:
We sure you want to check this in?  WIth a address?

Reply:
my bad...still editing haha